backend/src/test/java/it/com/crm/api/v1/CarControllerIntegrationTest.java

```diff
@@ -0,0 +1,88 @@
+package it.com.crm.api.v1;
+
+import com.crm.App;
+import com.crm.model.db.CarEntity;
+import com.crm.repository.CarRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = App.class)
+@AutoConfigureMockMvc
+@WithMockUser
+@Transactional
```

Comment:
I think its not needed

Reply:
Without this annotation second test does not pass, because on the database are still old data.